src/react/account/__tests__/AccountUserList.test.tsx

```diff
@@ -124,5 +140,88 @@ describe('AccountUserList', () => {
     expect(screen.getByText(SAMPLE_USER_NAME)).toBeInTheDocument();
 
     //TODO test copy arn
+    const firstRow = screen.getAllByRole('row')[1];
+
+    const arnButtonOfFirstRow = getAllByRole(firstRow, 'button');
+    expect(getByText(
+      arnButtonOfFirstRow[1],
+      /Copy ARN/i,
+    )).toBeInTheDocument();
+
+    fireEvent.mouseEnter(screen.getAllByRole('button')[4]);
+
+    expect(
+      screen.getByText('Copy ARN'),
+    ).toBeInTheDocument();
+
+    const eyeButtonOfFirstRow = getAllByRole(firstRow, 'button')[0];
+    expect(
+      eyeButtonOfFirstRow.getElementsByClassName('fas fa-eye')[0].className,
```

Comment:
ClassNames are not a reliable selector, please rely on accessible labels instead